docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md

```diff
@@ -110,34 +114,35 @@ The SAP Cloud Management service (technical name: `cis`) provides the Provisioni
 
     kubectl should return the list of Pods in the `default` namespace running in the cluster, which means that the cluster is accessible.
 
-> [!NOTE]
-> The following steps are optional and show how to revoke the credentials by deleting the binding.
+## Next Steps
 
-12. List all bindings for the instance.
+> [!NOTE]
+> For extra security, you can perform the optional steps to delete the binding sooner than it is set to expire in the **EXPIRATION_SECONDS** environment variable.
+> Otherwise, the binding is automatically deleted after the maximum allowed expiration time (7200 seconds) passes.
 
-    ```bash
-    curl -s "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings" -H "accept: application/json" -H "Authorization: bearer $TOKEN"
-    ```
+1.   (Optional) List all bindings for the instance.
 
-13. Delete the binding to revoke the credentials.
+      ```bash
+      curl -s "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings" -H "accept: application/json" -H "Authorization: bearer $TOKEN"
+      ```
 
-    ```bash
-    curl -s -X DELETE "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings/$BINDING_ID" -H "accept: application/json" -H "Authorization: bearer $TOKEN"
-    ```
+2.   (Optional) Delete the binding to revoke the credentials.
 
-    Try to access the cluster using kubectl. The connection should be refused, which means that the binding was successfully deleted and credentials revoked.
+      ```bash
+      curl -s -X DELETE "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings/$BINDING_ID" -H "accept: application/json" -H "Authorization: bearer $TOKEN"
+      ```
 
-    ```bash
-    kubectl get pods
-    ```
+      Try to access the cluster using kubectl. The connection should be refused, which means that the binding was successfully deleted and credentials revoked.
 
-## Next Steps
+      ```bash
+      kubectl get pods
+      ```
 
-To delete the instance bindings for the instance, perform steps 11 and 12. To deprovision the Kyma runtime, run:
+3.  To delete the instance bindings for the instance, perform steps 12 and 13. To deprovision the Kyma runtime, run:
```

Comment:
There are no 12 and 13 steps anymore. I think we can mention that the user can assign new binding ID to BINDING_ID env from the bindings list for another delete request.

Reply:
PK recommends against it, so let's skip it.

Reply:
ok.